src/Countries/US/NewJersey/NewJerseyDisabilityInsurance/V20190101/NewJerseyDisabilityInsurance.php

```diff
@@ -1,24 +1,31 @@
 <?php
 
-
 namespace Appleton\Taxes\Countries\US\NewJersey\NewJerseyDisabilityInsurance\V20190101;
 
-use Appleton\Taxes\Countries\US\NewJersey\NewJerseyDisabilityInsurance\NewJerseyDisabilityInsurance
-    as BaseNewJerseyDisabilityInsurance;
+use Appleton\Taxes\Countries\US\NewJersey\NewJerseyDisabilityInsurance\NewJerseyDisabilityInsurance as BaseNewJerseyDisabilityInsurance;
 use Illuminate\Database\Eloquent\Collection;
 
 class NewJerseyDisabilityInsurance extends BaseNewJerseyDisabilityInsurance
 {
+    const TAX_RATE = 0.0017;
     const WAGE_BASE = 34400;
-    const TAX_RATE = 0.005;
 
-    public function compute(Collection $tax_areas)
+    public function getBaseEarnings()
     {
-        return round($this->getAdjustedEarnings() * self::TAX_RATE, 2);
+        if (($this->payroll->earnings + $this->payroll->wtd_earnings) < self::WAGE_BASE) {
```

Comment:
can these use HasWageBase?

Reply:
Not the exact `getBaseEarnings` implementation. Should I be using `HasWageBase` anytime I use `getBaseEarnings` or am I missing something else?